lib/Model.js

```diff
@@ -569,6 +587,17 @@ Model.prototype.getPath = function getPath() {
     return arrayClone(this._path);
 };
 
+/**
+ * This one is actually private.  I would not use this without talking to
+ * jhusain, sdesai, or michaelbpaulson (github).
```

Comment:
I would add a warning out it potentially changing in future, and it being experimental